client-state-machine.go

```diff
@@ -821,6 +822,10 @@ func (state ClientStateWaitCV) Next(hr handshakeMessageReader) (HandshakeState,
 	} else {
 		logf(logTypeHandshake, "[ClientStateWaitCV] WARNING: No verification of server certificate")
 	}
+	serverCertChain := make([]*x509.Certificate, len(state.serverCertificate.CertificateList))
+	for i, certEntry := range state.serverCertificate.CertificateList {
+		serverCertChain[i] = certEntry.CertData
+	}
```

Comment:
Does the certificate authenticator do path construction?

Reply:
I'm not sure I understand your point. Certificate verification is broken anyway (#161).